aspnetcore/fundamentals/localization.md

```diff
@@ -232,7 +232,7 @@ Use `RequestLocalizationOptions` to add or remove localization providers.
 
 ## Resource file naming
 
-Resources are named for the type of their class minus the default namespace (which is also the name of the assembly). For example, a French resource in the `LocalizationWebsite.Web` project for the class `LocalizationWebsite.Web.Startup` would be named *Startup.fr.resx*. The class `LocalizationWebsite.Web.Controllers.HomeController` would be *Controllers.HomeController.fr.resx*. If for some reason your targeted class is in the same project but not in the base namespace you will need the full type name. For example, in the sample project a type `ExtraNamespace.Tools` would be *ExtraNamespace.Tools.fr.resx*.
+Resources are named for the full type name of their class minus the application name. For example, a French resource in the `LocalizationWebsite.Web` project for the class `LocalizationWebsite.Web.Startup` would be named *Startup.fr.resx*. The class `LocalizationWebsite.Web.Controllers.HomeController` would be *Controllers.HomeController.fr.resx*. If your targeted class's namespace is not the same as the application name you will need the full type name. For example, in the sample project a type `ExtraNamespace.Tools` would be *ExtraNamespace.Tools.fr.resx*.
```

Comment:
This should all be about the "assembly name" and not "application name".

Examples after the first need "a resource in the" and "would be _named_" verbiage.